docs/contributing/committing.md

```diff
@@ -15,84 +14,81 @@ Most code contributions will be in the form of pull requests from Github.
 
 Ideally, use this as an opportunity to squash the changes to a few commits, so
 each commit is making a single meaningful change (and not breaking anything).
 If this is not possible because of the nature of the changes, it's acceptable
 to either squash into a commit or leave all commits unsquashed,
 depending on which will be more readable in the commit history.
 
-.. code-block:: console
-
-    $ # Get the latest commits from Wagtail
-    $ git fetch upstream
-    $ git checkout main
-    $ git merge --ff-only upstream/main
-    $ # Rebase this pull request on to main
-    $ git checkout pr/xxxx
-    $ git rebase main
-    $ # Update main to this commit
-    $ git checkout main
-    $ git merge --ff-only pr/xxxx
-
-Update ``CHANGELOG.txt`` and release notes
-==========================================
-
-.. note::
-
-    This should only be done by core committers, once the changes have been reviewed and accepted.
-
-Every significant change to Wagtail should get an entry in the ``CHANGELOG.txt``,
+```console
+# Get the latest commits from Wagtail
+git fetch upstream
+git checkout main
+git merge --ff-only upstream/main
+# Rebase this pull request on to main
+git checkout pr/xxxx
+git rebase main
+# Update main to this commit
+git checkout main
+git merge --ff-only pr/xxxx
+```
+
+## Update `CHANGELOG.txt` and release notes
+
+```{note}
+This should only be done by core committers, once the changes have been reviewed and accepted.
+```
+
+Every significant change to Wagtail should get an entry in the `CHANGELOG.txt`,
 and the release notes for the current version.
 
-The ``CHANGELOG.txt`` contains a short summary of each new feature, refactoring, or bug fix in each release.
+The `CHANGELOG.txt` contains a short summary of each new feature, refactoring, or bug fix in each release.
 Each summary should be a single line.
 Bug fixes should be grouped together at the end of the list for each release,
 and be prefixed with "Fix:".
 The name of the contributor should be added at the end of the summary, in brackets.
 For example:
 
-.. code-block:: text
-
-     * Fix: Tags added on the multiple image uploader are now saved correctly (Alex Smith)
+```text
+* Fix: Tags added on the multiple image uploader are now saved correctly (Alex Smith)
+```
 
 The release notes for each version contain a more detailed description of each change.
 Backwards compatibility notes should also be included.
 Large new features or changes should get their own section,
 while smaller changes and bug fixes should be grouped together in their own section.
 See previous release notes for examples.
-The release notes for each version are found in ``docs/releases/x.x.x.rst``.
+The release notes for each version are found in `docs/releases/x.x.x.md`.
 
 If the contributor is a new person, and this is their first contribution to Wagtail,
-they should be added to the ``CONTRIBUTORS.rst`` list.
+they should be added to the `CONTRIBUTORS.md` list.
```

Comment:
This one is still rst for now. 
```suggestion
they should be added to the `CONTRIBUTORS.rst` list.
```
https://github.com/wagtail/wagtail/blob/main/CONTRIBUTORS.rst